docs/blog/2018-04-05-gatsby-video-tutorials/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Gatsby video tutorials a four part video series on Gatsby and Contentful by Khaled Garbaya
+date: 2018-04-05
+author: Carolyn Stransky
+canonicalLink: https://www.contentful.com/blog/2018/02/28/contentful-gatsby-video-tutorials/
+publishedAt: Contentful Blog
+excerpt: Video Tutorials for using Gatsby and Contentful
+---
+
```

Comment:
Could you add at the top in italics that the blog posts that they were originally posted on the Contentful blog?

Reply:
When you say at the top, do you mean in the title?

Reply:
Hey I made the changes in line 10